environment/README.md

```diff
@@ -96,6 +98,19 @@ Quality estimates are roughly
 | _Start Date_ | General date and time at which the _Gauge_ description was valid.
 | _End Date_ | General date and time at which the _Gauge_ description was no longer valid.
 
+#### _GTS_ ####
+
+| Field | Description |
+| --- | --- |
+| _Gauge_ | Code used to uniquely identify Tide Gauge _Stations_
```

Comment:
few questions about "names"
Does the gts file encompass Gauges and BPRs and is specific to dart sytems /BPR?  
Shall we make the difference between gauge and BPR ? 
and 
Should we  rename "tide gauge" tsunami gauge" ?

Reply:
The GTS and WMO numbers are just using a station so they aren't tied to a BPR and won't be updated every deployment 

We probably should move to Tsunami Gauge but as the tsunami gauges are called tide gauges throughout delta, it may need changing in a few places.